### DISCUSSION TITLE
How to delete Stages

### DISCUSSION THREAD

**Discussion by adarshmadrecha** [madrecha/jamku#605] [QA]
Suggested by `Thiruvachagan`

I have erroneously created stages for a service. Now I want to delete the same.

**Comment by Divwork92** [comment]
You can delete stages from service level.

Go to service page - click on the particular service for which you want to delete the stage 
From stage tab - click on delete icon which is on the right hand side  

Note*
Stages cannot be deleted if its linked with the tasks.
